csrc/apis/python/CMakeLists.txt

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+cmake_minimum_required(VERSION 3.14)
+project(mmdeploy_python)
+
+if (NOT TARGET pybind11)
+    add_subdirectory(${CMAKE_SOURCE_DIR}/third_party/pybind11 pybind11)
+endif ()
+
+pybind11_add_module(mmdeploy_python
+        ${CMAKE_CURRENT_SOURCE_DIR}/common.cpp
+        ${CMAKE_CURRENT_SOURCE_DIR}/classifier.cpp
+        ${CMAKE_CURRENT_SOURCE_DIR}/detector.cpp
+        ${CMAKE_CURRENT_SOURCE_DIR}/segmentor.cpp
+        ${CMAKE_CURRENT_SOURCE_DIR}/text_detector.cpp
+        ${CMAKE_CURRENT_SOURCE_DIR}/text_recognizer.cpp
+        ${CMAKE_CURRENT_SOURCE_DIR}/restorer.cpp)
+
+target_link_libraries(mmdeploy_python PRIVATE
```

Comment:
Can not link if `DMMDEPLOY_CODEBASES` is not `all`.

Reply:
fixed